### ISSUE TITLE
az storage fs exists: in case fs does not exist ResourceNotFoundError is returned

### ISSUE THREAD

**Issue by dr-dolittle** [Azure/azure-cli#15600]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
When checking for the existence of a filesystem in ADLS Gen2 via `az storage fs exists` (https://docs.microsoft.com/en-us/cli/azure/storage/fs?view=azure-cli-latest#az_storage_fs_exists) I get exists = True as response. In case it does not exist an error is returned

```
ResourceNotFoundError: The specified container does not exist.
RequestId:xxx-xxx-xxx-xxx-xxx
Time:2020-10-21T14:57:49.5062669Z
ErrorCode:ContainerNotFound
```

Please change that to return False in case the filesystem does not exist.

**To Reproduce**
Just run the command for both cases.

**Expected behavior**
True respectively False are being returned, not an exception.

**Environment summary**
Windows 10, azure-cli 2.13.0, PowerShell 5.1

**Comment by yonzhan** [comment]
add to S178

**Comment by dr-dolittle** [comment]
@yonzhan What is the current status on the issue? Is it already fixed respectively when will the fix be available for usage? Thanks

**Comment by Juliehzl** [comment]
Hi @dr-dolittle , it is already fixed in azure cli 2.15.0. Please update your version with ` az upgrade` and try again.
